src/common/single-line.js

```diff
@@ -0,0 +1,18 @@
+import React, { cloneElement } from 'react'
+import { propTypes } from 'utils'
+
+const SINGLE_LINE_STYLE = { display: 'flex' }
+const COL_STYLE = { margin: 'auto' }
+
+const SingleLine = propTypes({
```

Comment:
I'd like this component to be named `SingleLineRow`, because it must be clear this is a special kind or `Row` and it that it should be used like one.